src/pages/profile-page/profile-page.js

```diff
@@ -0,0 +1,312 @@
+                    className={classes.userInput}
+                    onChange={handleChange}
+                  />
+                </div>
+              </div>
+              <div className={classes.restOfUserInputs}>
+                <TextField
+                  label={PROFILE_DATA[language].email}
+                  name='email'
+                  value={user.email || ''}
+                  onChange={handleChange}
+                  className={classes.userInput}
+                />
+                <TextField
+                  label={PROFILE_DATA[language].phoneNumber}
+                  name='phoneNumber'
+                  value={user.phoneNumber ? user.phoneNumber : ''}
+                  onChange={handleChange}
+                  className={classes.userInput}
+                />
+                <TextField
+                  label={PROFILE_DATA[language].country}
+                  name='country'
+                  value={
+                    user.address
+                      ? user.address.country
+                        ? user.address.country
+                        : ''
+                      : ''
+                  }
+                  onChange={(e) => handleChange(e, 'address')}
+                  className={classes.userInput}
+                />
+                <TextField
+                  label={PROFILE_DATA[language].region}
+                  name='region'
+                  value={
+                    user.address
+                      ? user.address.region
+                        ? user.address.region
+                        : ''
+                      : ''
+                  }
+                  onChange={(e) => handleChange(e, 'address')}
+                  className={classes.userInput}
+                />
+                <TextField
+                  label={PROFILE_DATA[language].city}
+                  name='city'
+                  value={
+                    user.address
+                      ? user.address.city
+                        ? user.address.city
+                        : ''
+                      : ''
+                  }
+                  onChange={(e) => handleChange(e, 'address')}
+                  className={classes.userInput}
+                />
+                <TextField
+                  label={PROFILE_DATA[language].street}
+                  name='street'
+                  value={
+                    user.address
+                      ? user.address.street
+                        ? user.address.street
+                        : ''
+                      : ''
+                  }
+                  onChange={(e) => handleChange(e, 'address')}
+                  className={classes.userInput}
+                />
+                <TextField
+                  label={PROFILE_DATA[language].buildingNumber}
+                  name='buildingNumber'
+                  value={
+                    user.address
+                      ? user.address.buildingNumber
```

Comment:
Why do you have `user.address.buildingNumber` twice here?

Reply:
removed